docs/data/charts/composition/composition.md

```diff
@@ -13,11 +13,52 @@ packageName: '@mui/x-charts'
 ## Overview
 
 The `@mui/x-charts` follows an architecture based on context providers.
-The overall idea is to pass your series and axes definitions to a single component: the `<ChartContainer />`.
-This component transforms the data and makes it available to its children.
+The overall idea is to pass your series and axes definitions to special components.
+This component transforms the data and makes it available to its children, which can be composed.
 
-Based on the data provided by the container, you can render some graphical elements with provided subcomponents, such as `<LinePlot />` or `<ChartsYAxis />`.
-Or you can [create your own components](/x/react-charts/components/).
+There are two main classes of components, which are used to create a chart.
+
+### 1. Structural components
```

Comment:
```suggestion
### Structural components
```

Otherwise it looks weir in the menu

![image](https://github.com/user-attachments/assets/adbe3f00-73e3-448d-ac19-0261d4623e8b)